.github/workflows/build.yaml

```diff
@@ -249,6 +249,9 @@ jobs:
             packer/*.box
           if-no-files-found: error
   tests:
+    env:
+      VAGRANT_MEMORY: 3
+      VAGRANT_CPU: 10240
```

Comment:
well, macOS runners have 14GB, but keeping 4gb spare just in case... don't want the whole process to be OOMKilled without notice